.github/workflows/galata.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Install ipykernel pre-release that supports subshells (TEMPORARY)
         run: |
-          pip install --upgrade --pre ipykernel>=7
+          pip install --upgrade --pre ipykernel!=7.0.0a2
```

Comment:
Thanks 👍 

On other PRs, the UI tests now seem to be failing more often with this new version:

<img width="1982" height="520" alt="image" src="https://github.com/user-attachments/assets/4c719d2c-60ae-41e8-a94b-404bf1492bfb" />